Drawer/index.d.ts

```diff
@@ -0,0 +1,63 @@
+import MaterialComponent from '../MaterialComponent';
+import { VNode } from 'preact';
+import List from '../List/List';
+import { MDCFoundation, MDCComponent } from '../MaterialComponentsWeb';
+
+export default interface Drawer {
```

Comment:
This doesn't seem to work (TypeScript complains `'Drawer' only refers to a type, but is being used as a value here.` when trying to use it). I think it needs to be a class with static properties as all the other ones, even though the Drawer itself does not have any functionality.